CONTRIBUTING.md

```diff
@@ -73,14 +73,14 @@ _If you are interested in contributing code to Islandora but do not know where t
 
 In this case you should [browse open issues](https://jira.duraspace.org/issues/?jql=project%20%3D%20ISLANDORA%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20priority%20DESC).
 
-If you are contributing Drupal code, it must adhere to [Drupal Coding Standards](https://www.drupal.org/coding-standards); Travis CI will check for this on pull requests.
+If you are contributing Drupal code, it must adhere to [Drupal Coding Standards](https://www.drupal.org/coding-standards); Travis CI will check for this on pull requests. You should also follow the Islandora community's [Git Guidelines and Best Practices](https://github.com/Islandora/islandora/wiki/Git-Guidelines-and-Best-Practices). 
 
```

Comment:
The second link is contained within the other - and I was under the impression that the Committers Workflow is for Committers, with more detail than we expect from someone coming in for a first PR (and therefore getting all their info from contributing.md). I'll change it if folks think that's best, but my vote is to leave it as is.